src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,71 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { FormEvent, useEffect, useState } from 'react';
+import { CommentData } from '../types/Comment';
+
+type Props = {
+  handleOnAdd: (newComment: CommentData) => void,
+  isLoadingNewComment: boolean,
+  // isNameError: boolean,
+  // isEmailError: boolean,
+  // isBodyError: boolean,
+  skipAllErrors: boolean,
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  handleOnAdd,
+  isLoadingNewComment,
+  // isNameError,
+  // isEmailError,
+  // isBodyError,
+  skipAllErrors,
+}) => {
+  const [name, setName] = useState('');
+  const [isNameError, setIsNameError] = useState(false);
+  const [email, setEmail] = useState('');
+  const [isEmailError, setIsEmailError] = useState(false);
+  const [body, setBody] = useState('');
+  const [isBodyError, setIsBodyError] = useState(false);
+
+  const handleReset = () => {
+    setIsBodyError(false);
+    setIsEmailError(false);
+    setIsNameError(false);
+    setName('');
+    setEmail('');
+    setBody('');
+  };
+
+  useEffect(() => {
+    handleReset();
+  }, [skipAllErrors]);
+
+  const handleAdd = (event: FormEvent) => {
+    event.preventDefault();
+
+    const isNameEmpty = name.length === 0;
+    const isEmailEmpty = email.length === 0;
+    const isBodyEmpty = body.length === 0;
```

Comment:
its not enough because space is symbol and now I can create an empty comment, 
<img width="1599" alt="Screenshot 2023-03-01 at 14 12 37" src="https://user-images.githubusercontent.com/34283676/222136442-02b1d8a7-6896-4f5e-b597-a3b4ccc916bd.png">

Reply:
I also check it in work example, it possible to create such comment there

Reply:
@Malva37 will be better to improve solution and trim the initial value to disable such ability